pubspec.yaml

```
@@ -1,89 +1,35 @@
-  #       - asset: fonts/Schyler-Italic.ttf
-  #         style: italic
-  #   - family: Trajan Pro
-  #     fonts:
-  #       - asset: fonts/TrajanPro.ttf
-  #       - asset: fonts/TrajanPro_Bold.ttf
-  #         weight: 700
-  #
-  # For details regarding fonts from package dependencies,
-  # see https://flutter.dev/to/font-from-package
+  assets:
+    - shaders/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)